source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5843,9 +5829,9 @@ export class Column extends Container {
         return Enums.Orientation.Vertical;
     }
 
-    width: ColumnWidth = "auto";
+    width: ColumnWidth = "stretch";
```

Comment:
Question: Is this default behavior documented someplace where we will need to update our documentation as well with this change?

Reply:
Looks like the default is not doc'd here: [https://adaptivecards.io/explorer/Column.html](https://adaptivecards.io/explorer/Column.html). It probably should be.

Looks like shared model is defaulting to stretch as well.

---
In reply to: [342851467](https://github.com/microsoft/AdaptiveCards/pull/3571#discussion_r342851467) [](ancestors = 342851467)